Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/POSMessage.java

```diff
@@ -0,0 +1,32 @@
+package edu.asu.conceptpower.app.core;
+
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ResourceBundle;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+
+@Component
+@PropertySource("classpath:pos.properties")
+public class POSMessage {
```

Comment:
this seems overkill. you should be able to just use the messages right in the jsp file, similar to this:
http://stackoverflow.com/questions/11792734/accessing-dynamic-resource-message